Added prerequisites + Latch renewal period (#2833)
umbraco/UmbracoDocs: Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md

All new projects on Umbraco Cloud are automatically protected by Umbraco Latch. This means, that the default Umbraco Cloud URL for your project as well as any new hostnames you add will be assigned a TLS certificate automatically.

:::note
Certificates last 3 months and are automatically renewed, as long as the prerequisites below are met.
:::

## Prerequisites

> In order for Latch to work and the certificate to be applied the root of the site that the domain is pointing to cannot have the status of 404 whether it's a custom 404 or one caused by an error.

Reviewer: I'm not entirely sure I understand this sentence.
Is the point that, a domain cannot have a 404 status?

```suggestion
In order for Latch to work and the certificate to be applied to the root of the site that the domain is pointing to cannot have the status of 404 whether it's a custom 404 or one caused by an error.
```

> Latch needs the site to have a status of 200 in order for the certificate to be applied and renewed. This means that every 3 months Latch will go through the renewal process and the prerequisites have to be met each time in order for it to work.

Reviewer: ```suggestion
Latch needs the site to have a status of 200 in order for the certificate to be applied and renewed. This means that every 3 months Latch will go through the renewal process, and the prerequisites have to be met each time in order for it to work.
```

> If you wish to have a custom 404 page on the root of your website then you will need to purchase and apply a custom certificate instead.

Reviewer: Let's then add a link to the manual certificates article - what do you think?

```suggestion
If you wish to have a custom 404 page on the root of your website then you will need to purchase and apply a custom certificate instead.
```

Author: Yeah, sounds like a good idea. Can you add it there? I'm unsure on how to do is partway through the PR 😐

Reviewer: I believe the path is `../Manage-Hostnames/Security-Certificates/`.
Can add it I you like. Let me know!

:::

## Prerequisites

> In order for Latch to work and the TLS certificate to be applied to the site, you need to fullfill some criteria. First of all, the content node that the domain is pointing to cannot have the status of 404 whether it's a custom 404 page or one caused by an error.

Reviewer: ```suggestion
In order for Latch to work and the security certificate to be applied to the site, you need to fulfill some criteria. First of all, the content node that the domain is pointing to cannot have the status of 404 whether it's a custom 404 page or one caused by an error.
```
